sr_robot_commander/package.xml

```diff
@@ -24,6 +24,9 @@
   <build_depend>interactive_markers</build_depend>
   <build_depend>moveit_ros_planning</build_depend>
 
+  <build_depend>moveit_msgs</build_depend>
+  <run_depend>moveit_msgs</run_depend>
```

Comment:
can you put them with the others (new line between them, no new line before or after), roughly in the same order for build / run depend?